cypress/e2e/with-users/controllers/details.spec.ts

```diff
@@ -84,14 +84,12 @@ context("Controller details", () => {
   });
 
   it("displays controller commissioning details", () => {
+    cy.findAllByRole("gridcell", { name: "Name" })
```

Comment:
It feels like this should happen in the `beforeEach` rather than repeated in each test or is there a way we can get the controller id so we can `cy.visit()` the controller details page directly?

Reply:
I think it would make sense to use `cy.visit()` directly if we were creating a new controller via API call e.g. in `beforeAll`, but we have no way of doing that (yet). I moved the navigation to controller details page to `beforeEach` for now.